### ISSUE TITLE
Can I disable the SlimerJS default strict mode?

### ISSUE THREAD

**Issue by assafey** [laurentj/slimerjs#535]
### versions 

- SlimerJS: 0.906.2
- Firefox: 46.0.1
- Operating system: Mac, El-Capitan

### Steps to reproduce the issue

just navigate to [Wikipedia main page](http://en.wikipedia.org/wiki/Main_Page) with SlimerJS 

### Actual results:

Multiple javascript strict warnings

### Expected results:

No warnings

### Question:

Can I disable the SlimerJS default strict mode?

**Comment by wskorodecki** [comment]
Yes, just create `slimerjs/lib/slimer/defaults/preferences/prefs.js` file in slimerjs directory and add:
```javascript
pref("javascript.options.strict", false); // Show JS errors only.
```

**Comment by laurentj** [comment]
we could have a command line flag to enable/disable it

**Comment by chipironcin** [comment]
+1 to have a command line option to override slimerjs prefs from `defaults/preferences/prefs.js`

**Comment by sonickseven** [comment]
> Yes, just create `slimerjs/lib/slimer/defaults/preferences/prefs.js` file in slimerjs directory and add:
> 
> ```js
> pref("javascript.options.strict", false); // Show JS errors only.
> ```

i was searching this path a never i found it, my slimerjs directory is `/usr/local/lib/node_modules/slimerjs/`

**Comment by sonickseven** [comment]
> > Yes, just create `slimerjs/lib/slimer/defaults/preferences/prefs.js` file in slimerjs directory and add:
> > ```js
> > pref("javascript.options.strict", false); // Show JS errors only.
> > ```
> 
> i was searching this path a never i found it, my slimerjs directory is `/usr/local/lib/node_modules/slimerjs/`

## the file path is: 

`/usr/local/lib/node_modules/slimerjs/src/defaults/preferences/prefs.js`

And close this issue 😉
